tests/testthat/test-utils.R

```diff
@@ -362,3 +362,91 @@ test_that("remove_empty_rows works", {
     remove_empty_rows(my_record_id) %>%
     expect_equal(expected_out)
 })
+
+test_that("is_labelled works", {
+  df <- tibble::tribble(
+    ~"one", ~"two", ~"three",
+    1,        2,      3
+  )
+
+  is_labelled <- is_labelled(df)
+  expect_false(is_labelled)
+
+  labels <- c("One", "Two", "Three")
+  labelled::var_label(df) <- labels
+
+  is_labelled <- is_labelled(df)
+  expect_true(is_labelled)
+})
+
+test_that("make_skimr_labels works", {
```

Comment:
Let's come up with a better version of this test. In the current form it's trivial and also kind of brittle. One possibility would be to scrap the test altogether :)

Reply:
Yea, let's get rid of it. I was thinking more of how I didn't want to see our codecov badge take a hit haha.

Reply:
> Aaaah yes. Codecov 🙂
> How about a test that checks that the function returns a named list?

Sure sounds good to me! Gotta keep the number high and shiny